Source/Demo/MyObjejct.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include "CoreMinimal.h"
+#include "UObject/Object.h"
+#include "MyObjejct.generated.h"
+
+/**
+ * UMyObjejct is a Blueprintable interface for all the classes used in StringParseTool
+ */
+UCLASS(Blueprintable, BlueprintType, EditInlineNew)
+class DEMO_API UMyObjejct : public UObject
```

Comment:
I think you should remove StringParseTool from this project because it has a certain specific. This plugin is more about wide-using solutions. If you want to keep this logic for this plugin, it will be better to move this logic to the function library somehow